.semaphore/semaphore.yml.d/blocks/20-felix.yml

```diff
@@ -135,6 +171,8 @@
         value: 1.22.1
       - name: WINDOWS_VERSION
         value: "1809"
+      - name: FV_PROVISIONER
+        value: aws
     jobs:
       - name: VXLAN - Windows FV
```

Comment:
We should probably remove this, and use the CAPZ test for VXLAN coverage

Reply:
Will remove AWS VXLAN run in subsequent PR